File: Linear/Static/Array/03_Deleting/Given_Position/Built-In/Deleting.py
arr = [10, 20, 30, 40]

print("Array before deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")

pos = int(input("\nEnter a position: "))

# Delete the element at the given position.
del arr[pos - 1]

print("\nArray after deletion")
for i in range(len(arr)):
    print(arr[i], end=" ")

File: Linear/Static/Array/03_Deleting/Given_Position/Built-In/Deleting.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Scanner;

class DeletingGivenPositionBuiltIn {
    public static void main(String[] args) {
        ArrayList<Integer> arr = new ArrayList<>(Arrays.asList(10, 20, 30, 40));
        
        System.out.println("Array before deletion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }

        Scanner sc = new Scanner(System.in);
        System.out.println("\nEnter a position: ");
        int pos = sc.nextInt();

        // Delete the element at the given position.
        arr.remove(pos - 1);

        sc.close();

        System.out.println("\nArray after deletion");
        for (int i = 0; i < arr.size(); i++) {
            System.out.println(arr.get(i) + " ");
        }
    }
}
